src/components/SideBar/SideBarComponents/FacetFilters.js

```diff
@@ -77,8 +79,38 @@ const FacetPanel = ({ classes }) => {
     setGroupExpanded(groups);
   };
 
+  function selectionCounter(valueList) {
+    const selectedGroupNameIndex = selectedLabels.findIndex(
+      (selectedLabel) => selectedLabel.groupName === valueList[1],
+    );
+    const isChecked = !(valueList[3] === 'true');
+    if (selectedGroupNameIndex === -1) {
+      selectedLabels.push({ groupName: valueList[1], selectionCount: 1 });
```

Comment:
why don't you use a setState function, instead of pushing directly into state

Reply:
OK. I will update this by using the setState function